data/.gitignore

```diff
@@ -5,9 +5,3 @@ intermediate/acs/*.csv
 
 # Extra shapefile data
 intermediate/shapefiles/*.shp
-*/shapefiles/*.cpg
```

Comment:
We can still go ahead and ignore `intermediate/` shapefiles, I think.

Reply:
We should also remove the files from version control before merging, in addition to ignoring them.